apmpackage/apm/data_stream/traces.rum/elasticsearch/ingest_pipeline/apm_data_stream_migration.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
We don't need to duplicate this one.

Reply:
copy / paste issue, thanks for spotting.